src/main/java/org/opentest4j/AssertionFailedError.java

```diff
@@ -131,4 +119,11 @@ public ValueWrapper getActual() {
 		return this.actual;
 	}
 
+	@Override
+	public String toString() {
```

Comment:
Please document why `toString()` has been overridden and what it does with regard to _empty_ messages, noting the behavior of all public constructors with regard to how such messages are physically stored.